<?xml version="1.0"?>
<launch>
  <node name="plan2ctrl" pkg="motion_planner" type="motion_planner" output="screen" respawn="false">
  	<param name="action_mode" value="lasa_fixed"/>
  	<param name="wait_for_force" value="false" />
  	<param name="world_frame" value="/world_frame" />
	<param name="model_base_path" value="$(find motion_planner)/epfl-task-models/DoughRollingModels" />
  </node>
</launch>
